k8s/namespaces/wa/wa-task-batch-service/prod.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-service
+spec:
+  values:
+    java:
+      environment:
+        S2S_MICROSERVICE_NAME: test_service
+        WA_TASK_MONITOR_SERVICE_URL: http://wa-task-monitor-aat.service.core-compute-aat.internal
```

Comment:
Pointing to aat on prod ?

Reply:
it looks like the job was also deployed to prod and we want it to fail at the moment so it doesn't run. Just wanted to ensure it not talking to prod